core-data/src/commonMain/sqldelight/io/github/droidkaigi/confsched2022/data/TimetableItemSpeaker.sq

```diff
@@ -1,20 +1,20 @@
 CREATE TABLE timetableItemSpeaker (
-    timetableItemId TEXT NOT NULL PRIMARY KEY,
+    id TEXT NOT NULL PRIMARY KEY,
     name TEXT NOT NULL,
     iconUrl TEXT NOT NULL,
     bio TEXT NOT NULL,
     tagLine TEXT NOT NULL
 );
 
-CREATE INDEX timetableItemSpeaker_id ON timetableItemSpeaker(timetableItemId);
+CREATE INDEX timetableItemSpeaker_id ON timetableItemSpeaker(id);
 
 selectAll:
 SELECT *
 FROM timetableItemSpeaker;
 
 insert:
-INSERT INTO timetableItemSpeaker (
-    timetableItemId,
+INSERT OR IGNORE INTO timetableItemSpeaker (
+    id,
     name,
     iconUrl,
     bio,
```

Comment:
Does our application need migration? Can you implement this?
https://github.com/DroidKaigi/conference-app-2021/blob/23517099f70ce39c92202df4c9fd208f7740ce56/data/db/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/DestructiveMigrationSchema.kt#L6